packages/create-svelte/templates/skeletonlib/package.template.json

```diff
@@ -3,7 +3,8 @@
 	"version": "0.0.1",
 	"scripts": {
 		"dev": "vite dev",
-		"build": "svelte-kit sync && svelte-package",
+		"build": "vite build && npm run package",
+		"package": "svelte-kit sync && svelte-package",
```

Comment:
IIRC svelte-package doesn't do checking and svelte-check isn't included in the library template. svelte-check and a check command should probably be added in as well esp with #8484 merged.

Reply:
Good point. I wonder if we should try and make `svelte-check` only apply to packaged files rather than the entire app — would be annoying if you couldn't publish because of a finicky type error in your docs site that you don't really care about

Reply:
For that we could create a `tsconfig.lib.json` which only `include`s the files inside `lib` and reference that from `svelte-check`

Reply:
feels a bit messy to create a new file, and if `config.kit.files.lib` changed the developer would also need to change `tsconfig.lib.json`... perhaps there's a way to integrate checking into the `svelte-package` command instead?